scripts/dev/genfiles

```diff
@@ -35,6 +35,7 @@
 
 YACC=${YACC:-bison}
 YACC="$YACC -l"
+YFLAGS="-Wall"
```

Comment:
Add ``-Wextra`` maybe?

Reply:
These are flags for bison command-line tool: `bison -Wall ...`. The `-Wextra` doesn't exist.